.gitattributes

```diff
@@ -11,3 +11,6 @@
 
 # Some Windows-specific files should always be CRLF
 *.bat eol=crlf
+
+# Language detection
+PTVS/Python/Product/Miniconda/Miniconda3-x64/ linguist-vendored
```

Comment:
This needs to rooted at the repo, which means you shouldn't include `PTVS`. I also think you still need the wildcard at the end, i.e.:

```
Python/Product/Miniconda/Miniconda3-x64/* linguist-vendored
```

Reply:
You're right, fixing

Reply:
Done